### PR TITLE
Clean up the code.

### PR DESCRIPTION
**Pull request by yanggangtony**
/kind cleanup

Hey @everpeace .
I unintentionally search this repo from the k/k main repos.
I recently are learn and research the k8s code .

Just do some clean code for  [kube-throttler](https://github.com/everpeace/kube-throttler).
If this pr is make something mess , please feel free to told me..
Thanks.

### CODE REVIEW

File: pkg/scheduler_plugin/plugin.go [hunk 1]
```diff
@@ -112,7 +112,6 @@ func NewPlugin(configuration runtime.Object, fh framework.Handle) (framework.Plu
 	)
 
 	scheduleInformerFactory.Start(ctx.Done())
-	scheduleInformerFactory.WaitForCacheSync(ctx.Done())
```
Comment (everpeace) [inline pkg/scheduler_plugin/plugin.go:115]: 👍

File: pkg/scheduler_plugin/plugin.go [hunk 2]
```diff
@@ -249,7 +248,6 @@ func (pl *KubeThrottler) Unreserve(
 	if err != nil {
 		utilruntime.HandleError(errors.Wrapf(err, "Failed to unreserve pod %s/%s in ThrottleController", pod.Namespace, pod.Name))
 	}
-	pod.GetName()
```
Comment (everpeace) [inline pkg/scheduler_plugin/plugin.go:252]: 👍

**Review by everpeace — APPROVED** [review]
@yanggangtony No problem at all! Thank you very much for cleaning up my old code!

**Comment by yanggangtony** [comment]
Thanks

**Pull request merged**